### ISSUE TITLE
Working?

### ISSUE THREAD

**Issue by ronschaeffer** [michaelkkehoe/ha-linktap#1]
Hi there - Is this component working? If so, any set up instructions? I'd be happy to test.

**Comment by michaelkkehoe** [comment]
Yes it does. You can get an API token from here: https://www.link-tap.com/#!/api-for-developers

Add this repo to your HACS and restart Home Assistant and then add the integration.

You should be prompted for credentials. 

Apologies for being brief, but that should be enough to get started
